wordle-web/src/views/WordEditorView.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <v-dialog v-model="delDialog" persistent>
+    <v-card class="w-50 v-flex">
+      <v-card-text>Do you want to delete {{ delWord }}?</v-card-text>
+      <v-card-actions>
+        <v-btn color="red" @click="deleteWord">Delete</v-btn>
+        <v-btn color="grey" @click="delDialog = false">Cancel</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+  <v-dialog v-model="addDialog" persistent>
+    <v-card class="w-50">
+      <v-card-text>
+        <v-text-field label="Word" v-model="addWordText" />
+        <v-checkbox label="Is Common" v-bind="addWordIsCommon" />
+      </v-card-text>
+      <v-card-actions>
+        <v-btn color="green" @click="addWord">Add</v-btn>
+        <v-btn color="grey" @click="addDialog = false">Cancel</v-btn>
+      </v-card-actions>
+    </v-card>
+    <v-dialog v-model="duplicateDialog">
+      <v-card class="w-25">
+        <v-card-text>{{ addWordText }} already exists</v-card-text>
+        <v-card-actions>
+          <v-btn color="gray" @click="duplicateDialog = false">Ok</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </v-dialog>
+  <v-card class="mx-8 my-auto">
+    <v-row class="ma-3">
+      <v-col>
+        <v-text-field label="Search" v-model="searchPattern" @update:model-value="setViewList" />
+      </v-col>
+      <v-col>
+        <v-select
+          :items="pages"
+          v-model="currentPage"
+          label="Page"
+          @update:model-value="setViewList"
+        />
+      </v-col>
+    </v-row>
+    <v-row class="border ma-3">
+      <v-col class="text-h6">Word</v-col>
+      <v-col class="text-h6">Is Common</v-col>
+      <v-col>
+        <v-btn color="green" :disabled="!motu" @click="addDialog = true">Add Word</v-btn>
+      </v-col>
+    </v-row>
+    <v-row dense v-for="word in viewList" :key="word.text" class="border ma-3">
+      <v-col>{{ word.text }}</v-col>
+      <v-col><v-checkbox v-model="word.isCommon" @change="commonWord(word)" /></v-col>
+      <v-col><v-btn color="red" :disabled="!motu" @click="deleteDialog(word)">Delete</v-btn></v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script setup lang="ts">
+import Axios from 'axios'
+
+import { ListWord } from '@/scripts/listWord'
+import { ref } from 'vue'
+
+const motu = ref(true)
+
+const words = ref([
```

Comment:
The word editor should be accessing words from the database.  It should do a paginated search for all of the words matching the search, and changes should be reflected in real time.